Test/Machine/test_machine.py

```diff
@@ -345,6 +343,9 @@ def test_vector_jacobian():
 
 def test_log_val_diff():
```

Comment:
log_val_diff checks can be just removed, since we don't use this functionality anymore and python machines do not support it